inc/formatconvert.class.php

```diff
@@ -1516,11 +1618,13 @@ function computerSoftwareTransformation($a_inventory, $entities_id) {
                   $comp_key = strtolower($array_tmp['name']).
                                "$$$$".strtolower($array_tmp['version']).
                                "$$$$".$array_tmp['manufacturers_id'].
-                               "$$$$".$array_tmp['entities_id'];
+                               "$$$$".$array_tmp['entities_id'].
+                               "$$$$".$array_tmp['date_install'];
```

Comment:
Are you sure it's a good idea? Because you will have 2 different software if they are all same information except install_date